manageiq-operator/README.md

```diff
@@ -34,68 +27,70 @@ $ operator-sdk up local --namespace=<yournamespace>
 
 ### Option B: Run Inside Cluster
 
-First we need to install required Go packages. As `dep` is the dependency manager for this project, run
+Please refer to *"4. Build and run the Operator"* in [this guide](https://docs.openshift.com/container-platform/4.1/applications/operator_sdk/osdk-getting-started.html) for remaining steps
```

Comment:
This is kind of confusing. Can we just copy in the steps we need?

It should really just be building and pushing the image then creating the resources we need, right?

Reply:
How about handle this in future PRs? I want to introduce a `Makefile` for the operator to facilitate the build process.

Reply:
Is the build anything more than `operator-sdk build <image>`?

Reply:
Also the section is partially duplicated below. So I think we really should either put all the documentation here or deduplicate.

Really all this needs to be is:

- `operator-sdk build <image>`
- Update `deploy/operator.yaml` to use the new image tag
- Push the image
- Create stuff in deploy dir

Reply:
Even our "Option A" is already documented in the linked section.

Reply:
@Fryguy it will be just `oeprator-sdk build`, but I think the makefile should also include some other options, like build, test, linter, fmt etc.